cli/tests/unit/performance_test.ts

```diff
@@ -27,6 +27,12 @@ Deno.test(function timeOrigin() {
   assert(Date.now() >= origin);
 });
 
+Deno.test(function performanceToJSON() {
+  const json = performance.toJSON();
+
+  assert("timeOrigin" in json);
```

Comment:
It might be useful to also assert on:

1. There are no other keys.
2. The data type of `timeOrigin`.

Reply:
I have added more assertions!